docker/test/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+    nginx-reverse-proxy:
+        build:
+            context: .
+            dockerfile: nginx.Dockerfile
+        ports:
+            - "80:80"
+
+    irods-client-rest-cpp:
+        build:
+            context: .
+            dockerfile: irods_client_rest_cpp.Dockerfile
+        volumes:
+            - /home/alanking/hdd/builds/irods_client_packages/irods_client_rest_cpp/main/ubuntu-20.04:/packages:ro
```

Comment:
D'oh, forgot to remove my home directory here.

This is a provision until we release officially. Until then, we have to provide our own built packages when this is run. Alternatively, the built packages could/should be provided when the Dockerfile is built.

Reply:
I decided that we should install the local package at image build time. The one building the container must copy the package to the local build context, but I think this is much better than installing when the container starts. This also gives the benefit of being able to restart the REST service (which means we can update the configuration file in the mount point and restart the service to apply the changes).